vast.imp

```diff
@@ -0,0 +1,3 @@
+[
+  { ref: "caf.imp" }
+]
```

Comment:
Added to git by accident?

Reply:
This is an _iwyu-mapping-file_, together with the referenced `caf.imp` they contain out-of-band directives for telling the tool which header is to be used as the public source of a definition.